dockerunit-consul/pom.xml

```diff
@@ -3,7 +3,7 @@
 <modelVersion>4.0.0</modelVersion>
   <groupId>com.github.qzagarese</groupId>
   <artifactId>dockerunit-consul</artifactId>
-  <version>1.0.0-GA</version>
+  <version>1.0.1-GA-SNAPSHOT</version>
```

Comment:
May I ask why use the `GA` classifier? why not just `M.m.p-SNAPSHOT` for dev versions and `M.m.p` for releases?

Reply:
I have used this for `1.0.0` as opposed to the previous milestone releases `1.0.0.-Mx`,
but I'm fine with dropping `GA` moving forward.